HomeWork/HomeWork/Selenium WD/Pages/CompareProductPage.cs

```diff
@@ -0,0 +1,14 @@
+﻿using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+
+namespace HomeWork.Selenium_WD.Pages
+{
+    internal class CompareProductPage : BasePage
+    {
+        [FindsBy(How = How.XPath, Using = "//table[@id='compare_table']//child::a[contains(text(),'Apple iPad 2021')]")]
```

Comment:
Maybe better to use some abstract method for setting that ones? looks like elements with product name on this page is the same, except of 'text' parameter